ddhcpd/README.md

```diff
@@ -0,0 +1,32 @@
+# DDHCPD
+
+
+## Integration of DDHCPD in gluon
+
+Add the package `ddhcpd` in your `site.mk` and add a section in your `site.conf`:
+
+    ddhcpd = {
+      enabled = 1,
+      range = "10.187.124.0/22",
+    },
+
+Set enabled to `0` if you don't want DDHCPD to be enabled by default on your nodes.
+
+Choose a free IP-range that is not used by the DHCP-servers of your gateways
+that the DDHCPD can use to assign to clients.
+
+## Configuration
+
+Make sure your gateways use the same MAC address for their batman interface and
+the primary batman interface (which could also be the bridge) all the time.
+You need to install [mesh-announce](https://github.com/ffnord/mesh-announce) on
+your gateways, so DDHCPD can ask per respondd for the MAC address of the chosen
+gateway.
+
+#### disable DDHCPD with one shell call:
+
+     ssh <router ip> 'uci set ddhcpd.enabled="0"; uci commit ddhcpd && /etc/init.d/ddhcpd restart; echo done'
+
+## More
+
+A detailed documentation can be found at https://github.com/sargon/ddhcpd/blob/master/README
```

Comment:
More detailed documentation regarding DDHCPD itself can be found at https://github.com/sargon/ddhcpd/blob/master/README